src/freeze_model.py

```diff
@@ -87,7 +89,13 @@ def strip(input_graph, drop_scope, input_before, output_after, pl_name):
 
         config_train = config['config_train']
         config_train['xInput'] = config_train['feature_params']['xInput']
-        config_train['yInput'] = config_train['feature_params']['yInput']
+
+        feature_combination = 'audio_representation_dirs' in config_train
+
+        if feature_combination:
+            config_train['yInput'] = sum([i['yInput'] for i in config_train['features_params']])
+        else:
+            config_train['yInput'] = config_train['feature_params']['yInput']
```

Comment:
I don't understand why this change is necessary :)

Reply:
to recreate the architecture of the MPL before reloading the weights, `config_train['yInput']` has to be set to the right number of inputs. When the model was trained with a single type of embeddings, `config_train['feature_params']` is a dictionary where `yInput` is the embedding dimension. When the model was trained in a combination of embeddings, `config_train['feature_params']` is a list that can be iterated to get the number of dimensions of each embedding type. In this case `config_train['yInput']` is set to the sum of the number of dimensions for each embedding type.